test/framework/helper.go

```diff
@@ -25,8 +25,10 @@ import (
 	"testing"
```

Comment:
That should probably go upstream, isnt it ?

Reply:
yes true, it's already merged in upstream now :)